packages/dds/cell/src/cell.ts

```diff
@@ -191,7 +196,14 @@ export class SharedCell<T extends Serializable = any> extends SharedObject<IShar
             id: null,
         };
 
-        return tree;
+        // Return the snapshot tree and the list of routes tracked by the SnapshotSerializer.
+        return {
+            snapshot: tree,
+            routeDetails: {
+                source: this.id,
```

Comment:
I think this should be uniform across all layers, and be an array of node, routes.
I.e. not singleton, plural.
Key reason - future ready for data stores being on channels.

Reply:
What is the plan for SharedLogs that are too big for any single client to load into memory all at once?

(Right now, I snapshot them with modest amounts of memory by reusing previously uploaded blobs.)

Reply:
GC assumes it can fit the graph (with metadata) into memory to do its job - we are not building something more complex.
So it's responsibility of DDS to provide that data. It can (if it needs to) extract all references from prior blobs and store that metadata in separate blobs, such that only tis info needs to be paged in. For append-only DDS it's trivial - this metadata can also be organized as append-only log, such that only summarizer needs to load it and append new blobs on as needed bases.